src/domain/QuestionList/QuestionListData.tsx

```diff
@@ -0,0 +1,134 @@
+import { LoadingOutlined } from '@ant-design/icons';
+import { Input, Rate, Space, Spin } from 'antd';
+import { ColumnsType } from 'antd/lib/table';
+import { FilterConfirmProps } from 'antd/lib/table/interface';
+import React, { useRef } from 'react';
+
+import TableWrapper from '../../components/TableWrapper';
+import {
+  InterviewQuestionModel,
+  QuestionListModel,
+} from '../../services/Client';
+import {
+  createColumnFilterProps,
+  createColumnSearchProps,
+} from '../../services/table/tableUtils';
+import styles from './QuestionList.module.scss';
+
+export default function QuestionListData({
+  isBeingEdited,
+  list,
+  displayableQuestions,
+  addableQuestions,
+  allQuestions,
+  addToRemoveDrawerCallback,
+  addToAddDrawerCallback,
+}: {
+  isBeingEdited: boolean;
```

Comment:
I recommend move definition of props into separate file. It's hard to read code now.